rust/holochain_bootstrap_core/src/agent_info.rs

```diff
@@ -0,0 +1,184 @@
+//! Agent Info Structs
+
+use crate::decode::*;
+use crate::types::*;
+
+/// Struct for decoding agent info
+pub struct AgentInfoRef<'a> {
```

Comment:
is it possible/desirable to use the actual kitsune types here so they can always be kept in sync with versioning?

Reply:
Yeah, I was thinking kitsune would pull in these types so they'd be the same.

Reply:
@neonphog oh interesting, ok yeah we can do it that way